celery/utils/collections.py

```diff
@@ -3,7 +3,13 @@
 from __future__ import absolute_import, unicode_literals
 
 import sys
-import time
+
+try:
```

Comment:
Implementation and Python2/3 compatibility for the monotonic time function is already included in https://github.com/celery/vine/blob/1.0/vine/five.py#L63 , so perhaps it would be better to reuse this?